src/Expression/ForClasses/ResideInOneOfTheseNamespaces.php

```diff
@@ -32,7 +32,7 @@ public function evaluate(ClassDescription $theClass, Violations $violations, str
     {
         $resideInNamespace = false;
         foreach ($this->namespaces as $namespace) {
-            if ($theClass->namespaceMatches($namespace)) {
+            if ($theClass->namespaceMatches($namespace.'*')) {
```

Comment:
just out of curiosity, why do we need to add this?

Reply:
Its from a long time ago, but I guess it was because if a class is in a subnamespace is still in the given namespace.

However, looking at it now, it's out of scope fo this PR, and not having the `*` is a better option because we can match the namespace exactly or provide the namespace with a `*` which will match subnamespaces as well.

So, Im adding a fixup commit to remove this change.